server/src/setup/http/express/types.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express'
+
+export interface APIRouter extends express.Router {}
+
+export interface WebServer extends express.Express {}
+
+export interface Express {
```

Comment:
Pretty nit, but a more descriptive name would've made understanding the interactions easier. Up to you.

Reply:
I also dislike the name! The main problem is that anything with "App" in it conflicts with `application`, otherwise I'd totally call it `ExpressApp`.

How about `WaterparkExpress`?